merlin-rxjava/build.gradle

```diff
@@ -0,0 +1,38 @@
+apply plugin: 'com.android.library'
+apply plugin: 'bintray-release'
+
+dependencies {
+    compile project(':core')
+    compile 'com.android.support:support-annotations:25.3.1'
+    compile 'io.reactivex:rxjava:1.2.10'
+
+    testCompile 'org.easytesting:fest-assert-core:2.0M8'
+    testCompile 'org.mockito:mockito-core:1.9.5'
+    testCompile 'junit:junit:4.12'
+}
+
+tasks.withType(Test) {
+    scanForTestClasses = false
+    include "**/*Should.class"
+}
+
+android {
+    defaultConfig {
+        minSdkVersion 14
+        targetSdkVersion 25
```

Comment:
Referencing this issue in Dagger repo https://github.com/google/dagger/pull/591, we started to not having targetSdkVersion in libraries. 
Users may have lower target sdk version and this contributes to the manifest merger.